cg/meta/demultiplex/utils.py

```diff
@@ -95,3 +95,8 @@ def parse_flow_cell_directory_data(
 ) -> FlowCellDirectoryData:
     """Return flow cell data from the flow cell directory."""
     return FlowCellDirectoryData(flow_cell_path=flow_cell_directory, bcl_converter=bcl_converter)
+
+
+def append_flow_cell_name_to_fastq_file_path(fastq_file_path: Path, flow_cell_name: str) -> Path:
```

Comment:
Might be worth checking whether the flow cell name already is in the fastq file path. We could run into issues where we rename the file multiple times. It is called idempotence, we want to be able to run the flow multiple times and have the same result

Reply:
Maybe this is already done below 👍

Reply:
fixed